lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -131,6 +134,16 @@ def setup_students_and_grades(context):
                     )
 
 
+def unhide(unit):
+    """
+    Recursively unhide a unit and all of its children in the CCX
```

Comment:
I know this was a pre-existing function, but why recursion? A simple loop would do! :)